openapi/components/schemas/GatewayAccountConfig/dLocal.yaml

```
@@ -42,3 +42,8 @@ allOf:
             type: boolean
             default: false
             description: Specifies whether to create an installment plan and use it for payment.
+          customerDocumentCustomField:
+            type: string
+            description: |-
+              Name of the transaction custom field to use for sending customer document.
```

suggestion: I think this reads better if we place the adjective before the object 
```suggestion
              Name of the custom transaction field to use to send the customer document.
```

---

That seems incorrect that's not a custom transaction field, that's a field which is taken from transaction.customFields

---

I think the semantics changes.

`Custom field of the transaction` (customFields is a property on transaction resource) vs `Custom transaction field` (which looks like any propery on transaction)

---

I think your update on this description was perfect! Thanks for taking my input onboard 🙏